homeassistant/components/zha/core/channels/general.py

```diff
@@ -408,9 +408,9 @@ class OnOffConfiguration(ZigbeeChannel):
     """OnOff Configuration channel."""
 
 
-@registries.CLIENT_CHANNELS_REGISTRY.register(general.Ota.cluster_id)
 @registries.ZIGBEE_CHANNEL_REGISTRY.register(general.Ota.cluster_id)
-class Ota(ZigbeeChannel):
+@registries.CLIENT_CHANNELS_REGISTRY.register(general.Ota.cluster_id)
+class Ota(ClientChannel):
```

Comment:
CLIENT_CHANNELS_REGISTRY expects a ClientChannel, not a ZigbeeChannel.
I have therefore adjusted the inheritance, and inverted the decorators.

Is this correct?

Reply:
Yes, this is correct.